plugins/mcreator-core/procedures/item_istool.json

```diff
@@ -0,0 +1,50 @@
+{
+  "args0": [
+    {
+      "type": "input_value",
+      "name": "item",
+      "check": "MCItem"
+    },
+    {
+      "type": "field_dropdown",
+      "name": "tool_type",
+      "options": [
+        [
```

Comment:
Change these from `SwordItem` to `Sword`, for example, and add Item word in the template. This way this will be compatible with 1.12.2 in case someone wants to add this to 1.12.2 generator which uses names like `ItemSword`, not `SwordItem`

Reply:
It has been fixed!